File: Java/Java source-code/distance1.java


public class distance1 {
    public static double distance1(Double[] p1, Double[] p2) {
        double sum = 0;
        for (int i = 0; i < p1.length; i++) {
            sum += Math.abs(p1[i] - p2[i]);
        }
        return sum;
    }
}

File: Methods-C++/distance1.cpp
#include <iostream>
#include <math.h>
#include <algorithm>

double distance1(double p1[], double p2[], int n) {
    double sum = 0;
    for (int i = 0; i < n; i++) {
        sum += abs(p1[i] - p2[i]);
    }
    return sum;
}